ntoskrnl/ke/i386/cpu.c

```diff
@@ -1208,9 +1122,6 @@ KiIsNpxErrataPresent(VOID)
     INT ErrataPresent;
     ULONG Cr0;
 
-    /* Disable interrupts */
-    _disable();
```

Comment:
Why not necessary?

Reply:
This function is now called from `KiVerifyCpuFeatures`, which has interrupts disabled already

Reply:
I would typically indicate that with an `ASSERT(!(__readeflags() & EFLAGS_INTERRUPT_MASK))`